### PR TITLE
fix: refactor views display and add padding

### PR DESCRIPTION
**Pull request by wawrzonnn**

### CODE REVIEW

File: app/(profile)/create-profile/page.tsx [hunk 1]
```diff
@@ -16,7 +15,7 @@ const CreateProfilePage = async () => {
   }
 
   return (
-    <div className={styles.main}>
+    <div className={styles.container}>
```
Comment (Sumick) [inline app/(profile)/create-profile/page.tsx:18]: **nitpick:** This should be a separate component, eg: `Container`

File: src/components/Filters/Filters.tsx [hunk 2]
```diff
@@ -88,15 +88,9 @@ const Filters = () => {
...
-        <button className={styles.devTypeBtn + ' ' + styles.frontend}>
-          Frontend
-        </button>
-        <button className={styles.devTypeBtn + ' ' + styles.backend}>
-          Backend
-        </button>
-        <button className={styles.devTypeBtn + ' ' + styles.fullstack}>
-          Fullstack
-        </button>
+        <button className={styles.frontend}>Frontend</button>
```
Comment (Sumick) [inline src/components/Filters/Filters.tsx:91]: **nitpick:** I know it's not yours but I also should be separate component :D

File: app/globals.scss [hunk 3]
```diff
@@ -23,11 +23,7 @@ body {
 }
 
 .container {
-  max-width: 1180px;
-  min-height: 100vh;
+  // max-width: 1128px;
```
Comment (Sumick) [inline app/globals.scss:26]: **nitpick:** Container should always have maxWidth (which is specified in Figma) + it should be centered with `margin: 0 auto`

**Review by Sumick — CHANGES_REQUESTED** [review]
Few fixes

File: app/(profile)/my-profile/container/Container.tsx [hunk 4]
```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+import { PropsWithChildren } from 'react'
+import styles from './Container.module.scss'
+
```
Comment (Sumick) [inline app/(profile)/my-profile/container/Container.tsx:4]: **nitpick:** Why the same component is duplicated? We should have one container for centering web app content

**Review by Sumick — APPROVED** [review]
Approving, but please fix this duplicated container before merging

**Review by AniaPiWo — APPROVED** [review]

**Review by mimakuch — APPROVED** [review]
Looks cool!

**Pull request merged**